src/lib/marlin_plonk_bindings/stubs/src/bigint_384.rs

```diff
@@ -28,10 +28,10 @@ pub fn of_biguint(x: &BigUint) -> BigInteger384 {
 #[ocaml::func]
 pub fn caml_bigint_384_of_numeral(
     s: &[u8],
-    _len: u32,
-    base: u32,
+    _len: ocaml::Int,
+    base: ocaml::Int,
 ) -> Result<BigInteger384, ocaml::Error> {
-    match BigUint::parse_bytes(s, base) {
+    match BigUint::parse_bytes(s, base.try_into().unwrap()) {
```

Comment:
usually I would flag an unwrap within a function that returns a Result, but here returning a Result throws an exception anyway. Perhaps there is a difference because you can catch the exception at least? but not the unwrap?